modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -215,9 +214,10 @@ public function enqueue_scripts() {
 			'wpcomGutenberg',
 			array(
 				'switchToClassic' => array(
-					'isVisible' => false,
+					'isVisible' => $this->is_iframed_block_editor(),
+					'label'     => __( 'Switch to Classic Editor', 'jetpack' ),
```

Comment:
I wonder if this may turn out to be a bit confusing for self-hosted site owners. In our case, "classic editor" means the old Calypso editor. If you're the admin of a self-hosted site, it may mean the old wp-admin classic editor, usually available after you've installed the classic editor plugin.

It may be nothing though, I don't know. After all, I assume that the vast majority of site owners landing in the block editor in Calypso will be Atomic site owners.

Reply:
This label is only ever seen if the user is in an iframed editor on WordPress.com, so it'll always be in the context of the Calypso editor.